lib/components/ArtistScreen/artist_screen_content.dart

```diff
@@ -0,0 +1,87 @@
+import 'package:finamp/services/queue_service.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_sticky_header/flutter_sticky_header.dart';
+import 'package:get_it/get_it.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+
+import '../../models/finamp_models.dart';
+import '../../models/jellyfin_models.dart';
+import '../../services/jellyfin_api_helper.dart';
+import '../AlbumScreen/album_screen_content.dart';
+import '../AlbumScreen/song_list_tile.dart';
+import '../MusicScreen/music_screen_tab_view.dart';
+import 'albums_sliver_list.dart';
+
+class ArtistScreenContent extends StatefulWidget {
+  const ArtistScreenContent({Key? key, required this.parent}) : super(key: key);
+
+  final BaseItemDto parent;
+
+  @override
+  State<ArtistScreenContent> createState() => _ArtistScreenContentState();
+}
+
+class _ArtistScreenContentState extends State<ArtistScreenContent> {
+  Future<List<BaseItemDto>?>? songs;
+  Future<List<BaseItemDto>?>? albums;
+  JellyfinApiHelper jellyfinApiHelper = GetIt.instance<JellyfinApiHelper>();
+
+  @override
+  Widget build(BuildContext context) {
+    songs ??= jellyfinApiHelper.getItems(
+      parentItem: widget.parent,
+      filters: "Artist=${widget.parent.name}",
+      sortBy: "PlayCount",
+      includeItemTypes: "Audio",
+      isGenres: false,
+    );
+
+    return FutureBuilder(
+        future: songs,
+        builder: (context, songSnapshot) {
+          var orderedSongs = songSnapshot.data?.reversed.toList() ?? [];
+
+          albums ??= jellyfinApiHelper.getItems(
+            parentItem: widget.parent,
+            filters: "Artist=${widget.parent.name}",
+            sortBy: "ProductionYear",
+            includeItemTypes: "MusicAlbum",
+            isGenres: false,
+          );
+
+          return FutureBuilder(
```

Comment:
You could use something like `Future.wait()` to combine both futures, that seems a bit better than using two FutureBuilders...